### PR TITLE
Adds Gitpod Support

### PR DESCRIPTION
**Pull request by ijpatricio**
Here it is, a 1-click customizable installation of Filament's Demo project.

It's possible to use it inside the browser and with a local VSCode/Jetbrains. I forgot to do that in the video.

Let me know if a short (edited) YouTube video would be nice.

https://app.usebubbles.com/2Fhk4kzyqMnH1Sf4dg4XL9/filament-demo-on-gitpod

### CODE REVIEW

**Comment by ijpatricio** [comment]
Hey @danharrin 

So here it is as spoken. With the correct link, then I can proceed to the next step 😃

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ A demo application to illustrate how Filament Admin works.
 
 ![Filament Demo](https://user-images.githubusercontent.com/21066418/180368112-cfccda8d-a442-43e3-8cf3-3ba04f42247d.png)
 
+Try it right away, [open in Gitpod](https://gitpod.io/#https://github.com/filamentphp/demo)
```
Comment (danharrin) [inline README.md]:
```suggestion
[Open in Gitpod](https://gitpod.io/#https://github.com/filamentphp/demo) to edit it and preview your changes with no setup required.
```

**Comment by danharrin** [comment]
Still working?

**Comment by ijpatricio** [comment]
It will, after merge 😄

**Comment by ijpatricio** [comment]
Current main branch is missing the `.gitpod.yml` file.

**Pull request merged**